pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.387.1</jenkins.version>
+    <jenkins.version>2.426.1</jenkins.version>
```

Comment:
Needed because https://github.com/jenkinsci/jenkins/pull/7938 landed in 425 weekly